app/assets/stylesheets/_github-ribbon.css.sass

```diff
@@ -0,0 +1,48 @@
+// Responsive GitHub Ribbon
+
+.ribbon
+  background-color: #7d7d7d
+  overflow: hidden
```

Comment:
There are some weird behaviors around positioning with overflow. IIRC, you need to make sure the container it's within is set to a non-static positioning (i.e. `position: relative`).